pkgs/development/tools/language-servers/nixd/default.nix

```diff
@@ -0,0 +1,66 @@
+{ stdenv
+, meson
+, ninja
+, cmake
+, pkg-config
+, lit
+, nixUnstable
```

Comment:
I am actually very concerned that using `nixUnstable` will introduce some ABI incompatible updates from upstream. Maybe I should add a new nix package as the upstream of nixd?

Reply:
Just curious but which features on `nixUnstable` are required for nixd?

Reply:
> Just curious but which features on `nixUnstable` are required for nixd?

There is no particular need, but there are significant disruptive changes in command line argument parsing and path handling between nix stable and Unstable.

CommandLine parsing: https://github.com/nix-community/nixd/blob/b5079c4d79905048d3c0b39e1a2a6a66067f1111/lib/nixd/include/nixd/EvalDraftStore.h#L90